docs/project-contributions.md

```diff
@@ -42,49 +84,61 @@ pip install -U pytest
 python3 -m build --wheel
 ```
 
-Note: To peek at the Python code (uniffi generated and non-generated), run `maturin develop` and look in the c2pa folder.
+## Building wheels
```

Comment:
The above lines seems to duplicate this.  Which is correct?

Reply:
None. Running https://github.com/contentauth/c2pa-python/blob/main/Makefile#L49 does that locally. https://github.com/contentauth/c2pa-python/blob/main/Makefile#L85 verifies the builds.

But the release builds are only done by CI/CD now.

Reply:
(So the whole section likely can go - we could mention that you can test the wheel builds correctly locally by running the two commands mentioned above)